awswrangler/catalog/_utils.py

```diff
@@ -152,10 +177,25 @@ def sanitize_dataframe_columns_names(df: pd.DataFrame) -> pd.DataFrame:
     --------
     >>> import awswrangler as wr
     >>> df_normalized = wr.catalog.sanitize_dataframe_columns_names(df=pd.DataFrame({'A': [1, 2]}))
-
+    >>> df_normalized_drop = wr.catalog.sanitize_dataframe_columns_names(df=pd.DataFrame({'A': [1, 2], 'a': [3, 4]}), handle_dup_cols="drop")
+    >>> df_normalized_rename = wr.catalog.sanitize_dataframe_columns_names(df=pd.DataFrame({'A': [1, 2], 'a': [3, 4], 'a_1': [4, 6]}), handle_dup_cols="rename")
     """
     df.columns = [sanitize_column_name(x) for x in df.columns]
     df.index.names = [None if x is None else sanitize_column_name(x) for x in df.index.names]
+    if df.columns.duplicated().any():
```

Comment:
Nice

Reply:
This line is causing errors @jaidisido 
```
bash validate.sh                   
+ isort --check .
Skipped 3 files
+ black --check .
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
All done! ✨ 🍰 ✨
115 files would be left unchanged.
+ mypy --install-types --non-interactive awswrangler
awswrangler/catalog/_utils.py:211: error: "List[Any]" has no attribute "duplicated"
Found 1 error in 1 file (checked 71 source files)
```

Is there a way I can skip that in the testing?

Reply:
You can prefix your commits with `[skip ci]`. This will skip static code checking and tests.